File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/WCF/1033/SequentialWorkflowServiceLibrary/IWorkflow1.vb
' NOTE: You can use the "Rename" command on the context menu to change the interface name "IWorkflow1" in both code and config file together.
<ServiceContract()> _
Public Interface IWorkflow1

    <OperationContract()> _
    Function GetData(ByVal value As Integer) As String

    ' TODO: Add your service operations here

End Interface

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/WCF/1033/StateMachineWorkflowServiceLibrary/IWorkflow1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ServiceModel;

namespace $safeprojectname$
{
	// NOTE: You can use the "Rename" command on the "Refactor" menu to change the interface name "IWorkflow1" in both code and config file together.
	[ServiceContract]
	public interface IWorkflow1
	{

		[OperationContract]
		string GetData(int value);

		// TODO: Add your service operations here
	}
}
